secret/Tiltfile

```diff
@@ -1,13 +1,15 @@
 # -*- mode: Python -*-
 
-def secret_yaml_generic(name, namespace="", from_file=None):
+def secret_yaml_generic(name, namespace="", from_file=None, secret_type=None):
```

Comment:
i wish this could be `type=...`, but i see how that would conflict with the built-in operator!

Reply:
Yeah i firstly used `type` of course but Python does not really love that. :x